### DISCUSSION TITLE
buildspace: Build your own DAO with just Javascript

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#17] [Show and tell]
This week I'm going to spend times working on a project on buildspace. The project is about [Build your own DAO with just Javascript](https://zip.sc/LngHi). It's going to be an interesting project to understand how the DAO works. If somebody has already been working on this project or is interested in joining the project,, it would be nice to jump into a call to discuss or share feedback about the project together.

<a href="https://zip.sc/LngHi"><img src="https://user-images.githubusercontent.com/131172/146790765-1c3db714-d69e-449f-b775-8b9b5312fc0a.png " width="400"></a>

**Comment by BunHouth** [comment]
That such as a great resource for learn and build an real product.

**Reply by samnang** [reply]
Do you want to jump into a call on Thursday to discuss this project together?

**Reply by BunHouth** [reply]
On Thursday after working hour, I'm available to do that. I'm available from 5pm Cambodia time.

**Reply by samnang** [reply]
Perfect, let do it at 5:30pm then.

**Comment by saingsab** [comment]
Wonderful, I will definitely join.

**Comment by samnang** [comment]
Here is the meeting link to join: https://meet.jit.si/moderated/f2125224db9d06cb05208e4602d1e13def2efe342e09e1bcd3e25589632fa218

We will start at 5:30pm.
